dao.py

```diff
@@ -0,0 +1,40 @@
+import os
+from pymongo import MongoClient
+from bson.objectid import ObjectId
+
+DB_NAME = 'schedule_practice'
+
+class DAO:
+    __db = MongoClient(os.environ['DB_PORT_27017_TCP_ADDR'], 27017)[DB_NAME]
+
+    """
+    Read Operations
+    """
+    def _get_by_id(self, collection, id_str):
+        return self.__db[collection].find_one({'_id': ObjectId(id_str)})
+
+    def _get_many_by_params(self, collection, selector):
+        return self.__db[collection].find(selector)
+
+    def _get_one_by_params(self, collection, selector):
```

Comment:
not needed, this method just returns the first occurrence of the parameter request .. who is gonna use that!?